proto/proto.go

```diff
@@ -163,40 +164,64 @@ type JobLog struct {
 	Stderr string `json:"stderr"` // stderr output
 }
 
+type JobLogById []JobLog
+
+func (jls JobLogById) Len() int { return len(jls) }
+func (jls JobLogById) Less(i, j int) bool {
+	return jls[i].RequestId+jls[i].JobId < jls[j].RequestId+jls[j].JobId
+}
+func (jls JobLogById) Swap(i, j int) { jls[i], jls[j] = jls[j], jls[i] }
+
 // JobStatus represents the status of one job in a job chain.
 type JobStatus struct {
-	RequestId string                 `json:"requestId"`
-	JobId     string                 `json:"jobId"`
-	Type      string                 `json:"type"`
-	Name      string                 `json:"name"`
-	Args      map[string]interface{} `json:"jobArgs"`
-	StartedAt int64                  `json:"startedAt"` // when job started (UnixNano)
-	State     byte                   `json:"state"`     // usually proto.STATE_RUNNING
-	Status    string                 `json:"status"`    // real-time status, if running
-	Try       uint                   `json:"try"`       // try number, can be >1+retry on sequence retry
+	RequestId string `json:"requestId"`
+	JobId     string `json:"jobId"`
+	Type      string `json:"type"`
+	Name      string `json:"name"`
+	StartedAt int64  `json:"startedAt"`        // when job started (UnixNano)
+	State     byte   `json:"state"`            // usually proto.STATE_RUNNING
+	Status    string `json:"status,omitempty"` // real-time status, if running
+	Try       uint   `json:"try"`              // try number, can be >1+retry on sequence retry
 }
 
-// JobChainStatus represents the status of a job chain reported by the Job Runner.
-type JobChainStatus struct {
-	RequestId   string      `json:"requestId"`
-	JobStatuses JobStatuses `json:"jobStatuses"`
-}
+// JobStatusByStartTime sorts []JobStatus by StartedAt ascending (oldest jobs first).
+type JobStatusByStartTime []JobStatus
 
-// RequestStatus represents the status of a request reported by the Request Manager.
-type RequestStatus struct {
-	Request        `json:"request"`
-	JobChainStatus JobChainStatus `json:"jobChainStatus"`
-}
+func (js JobStatusByStartTime) Len() int           { return len(js) }
+func (js JobStatusByStartTime) Less(i, j int) bool { return js[i].StartedAt < js[j].StartedAt }
+func (js JobStatusByStartTime) Swap(i, j int)      { js[i], js[j] = js[j], js[i] }
 
 // RequestProgress updates request progress from the Job Runner.
 type RequestProgress struct {
 	RequestId    string `json:"requestId"`
 	FinishedJobs uint   `json:"finishedJobs"` // number of jobs that ran and finished with state = STATE_COMPLETE
 }
 
+// RunningStatus represents running jobs and their requests. It is returned by
+// GET /api/v1/status/running.
```

Comment:
Specify that it's returned by the Request Manager from /api/v1/status/running, since the JR has the same endpoint.

Reply:
Done.